src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx

```diff
@@ -30,7 +31,16 @@ Our New Relic Flutter agent monitors your Flutter mobile app and provides deep i
 
 ## (Recommended) Guided installation [#guided-install]
 
-To install the Flutter agent, follow our [guided install](https://onenr.io/0qQagg4AGR1), located directly in the UI.
+To install the Flutter agent, follow the guided install:
+1. Go to **[one.newrelic.com](https://one.newrelic.com/) > Integrations & Agents**.
+2. Search for "Flutter" and click one of the tiles:
```

Comment:
You may want to check to see if you can link directly to Flutter. Does this link work for you? https://onenr.io/0OQMGB34eRG

Reply:
@paperclypse Ah yes, this is a good callout. Thank you!